.github/renovate.json5

```diff
@@ -371,11 +374,28 @@
         "on monday"
       ],
     },
+    {
+      "groupName": "all kind-images main",
+      "groupSlug": "all-kind-images-main",
```

Comment:
@aanm these names seem to imply they only apply to the `main` branch, but this definitely seems to be applying to all branches. For example: 

https://github.com/cilium/cilium/pull/27772

Is that what `main` is supposed to mean here? Should this section have a stanza to restrict the `matchBaseBranches` to `main`?

Reply:
@joestringer No, I think the "main" was a bad copy paste.